web-server/v0.4/src/routes/Dashboard/RunComparison.js

```diff
@@ -38,6 +56,16 @@ export default class RunComparison extends ReactJS.Component {
       graphKeys: [],
       tableData: [],
       loading: false,
+      generatingPdf: false,
+      modal: {
+        visible: false,
+      },
+      checked: ['details', 'summary', 'table', 'timeseries'],
+      defaultChecked: [],
+      pdf: {
+        pdfHeader: ' ',
+        confirmLoading: false,
+      },
```

Comment:
We'll also want add a state item for `pdfName`.

Reply:
Added.